src/Service/HealthCheck/DabHealthCheckReport.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+
+namespace Azure.DataApiBuilder.Config.ObjectModel
+{
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum HealthStatus
+    {
+        Healthy,
+        Unhealthy
+    }
+
+    /// <summary>
+    /// The health report of the DAB Engine.
+    /// </summary>
+    public record DabHealthCheckReport
```

Comment:
nit: this is in Dab repo, so probably no need to prefix with Dab again, could simply be HealthCheckReport